etc/config-template.toml

```diff
@@ -58,7 +58,8 @@
 ## Max running tasks of each worker, reject if exceeded.
 # max-tasks-per-worker = 2000
 
-[readpool.storage]
+# Uncomment this section if you don't intend to unify the read pools.
```

Comment:
The comment is confusing. What if I keep both this section and unify thread pool section?

Reply:
I don't have a good idea about how to know whether the user has configured the old thread pools.